Quick_Deploy/vLLM/README.md

```diff
@@ -38,36 +38,18 @@ Triton Inference Server using Triton's [Python backend](https://github.com/trito
 [known limitations](#limitations).
 
 
-## Step 1: Build a Custom Execution Environment With vLLM and Other Dependencies
+## Step 1: Build a Triton Container Image with vLLM
 
-Running vLLM within Triton container requires us to provide
-[custom execution environment](https://github.com/triton-inference-server/python_backend#creating-custom-execution-environments)
-with vLLM and other package dependencies. The provided script should build the package environment
-for you which will be used to load the model in Triton.
+We will build a new container image derived from tritonserver:23.08-py3 with vLLM.
 
 ```
-docker run --gpus all -it --rm -p 8000:8000 -p 8001:8001 -p 8002:8002 --shm-size=8G --ulimit memlock=-1 --ulimit stack=67108864 -v ${PWD}:/work -w /work nvcr.io/nvidia/tritonserver:23.08-py3 bash
-./gen_vllm_env.sh
+docker build -t triton_vllm .
 ```
 
-This step might take a while to build the environment packages. Once complete, the provided sample
-[model_repository](model_repository) will be populated with
-`triton_python_backend_stub` and `vllm_env`.
+The above command should create triton_vllm image with vLLM and all of its dependencies.
 
-## Step 2: Set Up Triton Inference Server
 
-The structure of the sample model repository as follows after the Step 1 above:
-```
-model_repository/
-`-- vllm
-    |-- 1
-    |   `-- model.py
-    |-- config.pbtxt
-    |-- triton_python_backend_stub
-    |-- vllm_engine_args.json
-    `-- vllm_env
-
-```
+## Step 2: Start Triton Inference Server
```

Comment:
I think it would be a good idea to display structure of model_repository anyways.

Reply:
Sounds good, added.